saleor/attribute/migrations/0038_remove_assignedproductattribute.py

```diff
@@ -0,0 +1,108 @@
+# Generated by Django 3.2.20 on 2023-07-11 16:10
+
+import django.db.models.deletion
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("attribute", "0037_remove_assignedpageattribute"),
+    ]
+
+    state_operations = [
+        migrations.RemoveField(
```

Comment:
Also, do we have a task for removing those tables from DB?

Reply:
https://github.com/saleor/saleor/issues/14594 🫡